frontend/src/routes/+page.svelte

```diff
@@ -3,6 +3,22 @@
   import TgHeadingMain from "$lib/components/base/TgHeadingMain.svelte";
   import TgHeadingTitle from "$lib/components/base/TgHeadingTitle.svelte";
   import TgParagraph from "$lib/components/base/TgParagraph.svelte";
+  import TgLink from "$lib/components/base/TgLink.svelte";
+  import { get } from "$lib/services/rest";
+  import { onMount } from "svelte";
+
+  var status = true;
+
+  onMount(async () => {
+    try {
+      const res = await get("status");
+      if (res.status !== "UP") {
+        status = false;
+      }
+    } catch (err) {
+      console.error("err: ", err);
```

Comment:
An error should also set the status variable to `false`. In fact, the status endpoint will literally never return anything other than `UP` unless there is an error (but I would still keep the `if (res.status !== "UP")` check in case the status endpoint is expanded).

Reply:
It almost sounds like I can get rid of the try/catch due to the nature of the endpoint, is that what you were suggesting?

Reply:
No, I think you need the try/catch because if the service is actually down or throttled then the gateway will return something like a 429 or 504 status code and those will [cause `get` to throw an Error](https://github.com/skill-collectors/guesstimator/blob/43a3f4aeacdc487599e48d582143555b429b4a08/frontend/src/lib/services/rest.ts#L140).

If anything you could get rid of the `if (res.status !== "UP") check, but I'm okay leaving it in. If nothing else it expresses to someone reading the code what a good response is expected to look like.